flake.nix

```diff
@@ -127,6 +127,7 @@
               # to be generated
               gcc
               tree-sitter
+              docgen
```

Comment:
I prefer to run docgen from within the shell then `nix run .#docgen` which is slower (it copies the fodler to the store + nix eval time).